backend/fastrtc/webrtc_connection_mixin.py

```diff
@@ -80,12 +85,32 @@ def __init__(self):
         self.modality: Literal["video", "audio", "audio-video"]
         self.mode: Literal["send", "receive", "send-receive"]
         self.allow_extra_tracks: bool
+        self.rtc_configuration: dict[str, Any] | None | RTCConfigurationCallable | None
+        self.server_rtc_configuration: RTCConfiguration | None
 
     @staticmethod
     async def wait_for_time_limit(pc: RTCPeerConnection, time_limit: float):
         await asyncio.sleep(time_limit)
         await pc.close()
 
+    @staticmethod
+    def convert_to_aiortc_format(
+        rtc_configuration: dict[str, Any] | None,
+    ) -> RTCConfiguration | None:
+        rtc_config = rtc_configuration
+        if rtc_config is not None:
+            rtc_config = RTCConfiguration(
+                iceServers=[
```

Comment:
This is not a bug but just let me note that,
this will result in `rtc_config.iceServers = []` when the passed `rtc_configuration.iceServers is None`, and `aiortc` behaves differently for `[]` and `None` as it uses the [default STUN server if `iceServers is None`](https://github.com/aiortc/aiortc/blob/9664b669bcee50dc41e341656b2f8ab19ce69623/src/aiortc/rtcicetransport.py#L180-L181) while the ICE servers will be empty if it's set as `[]`.
IMO it won't be a problem because nothing will be changed unless the user sets `server_rtc_configuration` explicitly though, some documentation may be helpful?

Reply:
Noted - good catch!